contrib/DC-GAN.ipynb

```diff
@@ -0,0 +1,484 @@
+{
```

Comment:
This is missing the Colab setup.
This is a requirement for our Colabs.

Also, all the outputs should be stripped.

Reply:
Done. Pushed in f3956b5.

Reply:
moved.